fendermint/app/src/cmd/genesis.rs

```diff
@@ -280,6 +283,19 @@ fn set_ipc_gateway(genesis_file: &PathBuf, args: &GenesisIpcGatewayArgs) -> anyh
     })
 }
 
+async fn seal_state(genesis_file: &PathBuf, args: &SealGenesisArgs) -> anyhow::Result<()> {
+    let genesis = read_genesis(genesis_file)?;
+
+    let genesis_creator = GenesisCreator::new(
```

Comment:
Maybe `GenesisStateCreator`?

Reply:
I think should be fine with `Genesis`, I renamed everything else to align with `genesis` and removed `state`.